bastion_current_ip.py

```diff
@@ -0,0 +1,27 @@
+# obtain current public ip, create a variables.tfvars file 
+# with this ip as the user_ip (along with other static data). 
+# Run terraform shell script to apply the instance
+
+import os,subprocess
+
+myip = subprocess.check_output(["dig","+short","myip.opendns.com","@resolver1.opendns.com"])
```

Comment:
You've got a good function here for getting your IP.  You should put this in a function so you can use it other places.